modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/java/com/liferay/frontend/taglib/clay/sample/web/internal/display/context/ClaySampleNavigationCard.java

```diff
@@ -48,7 +48,7 @@ public String getImageAlt() {
 
 	@Override
 	public String getImageSrc() {
-		return "https://images.unsplash.com/photo-1502290822284-9538ef1f1291";
+		return "";
```

Comment:
Maybe this change came to avoid an error. I don't have the whole context as @nhpatt started the task.

I'll change it to use a local resource that is also committed in the repo

Reply:
If we are creating a new module, we can just leave the `frontend-taglib-clay-sample-web` as it is, and improve this in a separate task.